### ISSUE TITLE
Use media_kit for desktop playback support?

### ISSUE THREAD

**Issue by Quackdoc** [lamarios/clipious#48]
It would be nice to have desktop support using media_kit. I am leaving this as a request, using media_kit for desktop is nice since it currently supports windows and linux with someone working on an osx PR. 

It has very good performance, and supports a large chunk of mpv's features under the hood with some more advanced work. the main features would get out of it are Split audio/video support so greater then 1080p resolutions. and hwdecode support on desktop platforms. 

This is something I may try to implement myself sometime in the future, however I don't have much time lately. I do have a similar PR up for pstube in which case basic groundwork is done (though it uses piped api) 

if you would like this feature, don't plan on implementing it, and don't mind waiting, at some point in time I will be able to work on it.

[Media Kit Repo](https://github.com/alexmercerind/media_kit/)
[Link to Pstube PR](https://github.com/prateekmedia/pstube/pull/86)

**Comment by lamarios** [comment]
Yeah I agree it would be nice. though I'm not rushing as Android is my primary target for now. iOs already has Yattee that is good and I don't want to pay the $99 / year anymore. 

Sure, i'll be waiting for your PR :) 

Thanks !
